### PR TITLE
New admin tab displays server env vars + system properties

### PR DESCRIPTION
**Pull request by amcclain**
+ Requires corresponding hoist-core update to enable, disables gracefully otherwise.
+ Add brief intro labels to other server-side tabs for clarity.

Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by amcclain** [comment]
Goes with https://github.com/xh/hoist-core/pull/219

Hi @lbwexler - this seems like it would be useful, and is a minimal addition. I had a few small discussion items we can review on it whenever we have a chance.

**Comment by lbwexler** [comment]
Looks good, happy to discuss!

File: admin/tabs/server/environment/ServerEnvPanel.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    item: 'Server-side environment variables and JVM system properties',
+                    className: 'xh-bold'
+                }),
+                filler(),
+                gridCountLabel({unit: 'entries'}),
+                '-',
+                storeFilterField({matchMode: 'any'}),
+                exportButton()
+            ],
+            item: lastLoadException ? errorMessage({error: lastLoadException}) : grid(),
```
Comment (lbwexler) [inline admin/tabs/server/environment/ServerEnvPanel.js:32]: this is the pattern that makes you wish we had that error property on panel.   Wonder if we just needed an additional flag to say whether toolbars should be shown or not?.....

File: admin/tabs/server/environment/ServerEnvModel.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    @managed gridModel;
+
+    get hasMinHoistCoreVersion() {
+        return checkMinVersion(XH.environmentService.get('hoistCoreVersion'), '10.1');
+    }
+
+    constructor() {
+        super();
+
+        const {STRING} = FieldType;
```
Comment (lbwexler) [inline admin/tabs/server/environment/ServerEnvModel.js:19]: nit:  I find this distracting as the second line in this method -- an utterly trivial thing getting in the way of the main line of code, and with caps.  

Would we consider putting it at the level of the file, say, right after the imports on line 6?

**Pull request merged**